core/src/main/java/org/apache/iceberg/rest/ResourcePaths.java

```diff
@@ -74,4 +74,16 @@ public String table(TableIdentifier ident) {
   public String rename() {
     return SLASH.join("v1", prefix, "tables", "rename");
   }
+
+  public String scanReport(TableIdentifier identifier) {
+    return SLASH.join(
+        "v1",
+        prefix,
+        "namespaces",
+        RESTUtil.encodeNamespace(identifier.namespace()),
+        "tables",
+        RESTUtil.encodeString(identifier.name()),
+        "metrics",
+        "scan");
```

Comment:
Is there a reason to make multiple endpoints? Why not just post to metrics with a type included in the serialized metrics report?

Reply:
see my comment in https://github.com/apache/iceberg/pull/5407#discussion_r940060609 about the initial reasoning for a `/metrics/scan` endpoint (rather than just `/metrics`).
Obviously we could post simply to `/metrics` with a type but eventually you might want to also do a `GET` and retrieve metrics of a particular type, so using `/metrics/scan` seems more in-line with how REST would model post/get requests.

Example: We could have a `/metrics/commit` endpoint where we'd `POST` commit metrics to and could also perform a `GET` to retrieve them

I understand the reasoning for this, since we'd like to keep changes to the REST spec to a minimum. I don't have a strong opinion on this, but I'd like us to at least consider this. IMO this is a bit cleaner and aligns better with REST to have it this way. However, I'm interested in what others think about this

Reply:
@danielcweeks, what do you think about having one metrics endpoint vs. separate endpoints for different metrics types?

Reply:
I thought about this for a while and I think we should go with the generic metrics endpoint rather than specific endpoints for scan, commit, etc. That keeps the REST spec simpler. I also think that it fits with the potential for future GET requests because we could provide a listing of metric reports and filter them using query params for the `.../metrics` endpoint. We could also return individual reports using `.../metrics/<report-id>`.